### ISSUE TITLE
[Package Request]: Microsoft.UI.Xaml

### ISSUE THREAD

**Issue by denelon** [microsoft/winget-pkgs#111840]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/microsoft/microsoft-ui-xaml
Publisher: Microsoft
Package Name: Windows UI Library
Description: WinUI is a user interface layer that contains modern controls and styles for building Windows apps.
Package Versions: 2.7.3 & 2.8.5 
Installer URLs:

https://github.com/microsoft/microsoft-ui-xaml/releases/tag/v2.7.3
https://github.com/microsoft/microsoft-ui-xaml/releases/tag/v2.8.5

I'd propose the ID: Microsoft.UI.Xaml
Moniker: winui

Related to:
* https://github.com/microsoft/winget-cli/issues/2134

**Comment by denelon** [comment]
The upgrade behavior should be "install" so side-by-side installation can work correctly, or this should be treated as separate packages like other frameworks/libraries.

**Comment by Trenly** [comment]
@denelon - Should be resolved now

```
winget install Microsoft.UI.Xaml.2.7
winget install winui-2.7

winget install Microsoft.UI.Xaml.2.8
winget install winui-2.8
```

**Comment by cbr-github** [comment]
@denelon and @Trenly :

When installing

https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/UI/Xaml/2/8

I'm getting the following error:
```
Installation wird ausgeführt...
-
███▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  1024 KB / 8.44 MB
███████▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  2.00 MB / 8.44 MB
██████████▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  3.00 MB / 8.44 MB
██████████████▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  4.00 MB / 8.44 MB
█████████████████▒▒▒▒▒▒▒▒▒▒▒▒▒  5.00 MB / 8.44 MB
█████████████████████▒▒▒▒▒▒▒▒▒  6.00 MB / 8.44 MB
████████████████████████▒▒▒▒▒▒  7.00 MB / 8.44 MB
████████████████████████████▒▒  8.00 MB / 8.44 MB
██████████████████████████████  8.44 MB / 8.44 MB
-
Found Microsoft.UI.Xaml [Microsoft.UI.Xaml.2.8] Version 8.2306.22001.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Successfully verified installer hash
Starting package install...
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  2%
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  2%
████████████████████████████▒▒  95%
Installer failed with exit code: 0x80070002 : Das System kann die angegebene Datei nicht finden.
```

This package is required for Terminal version:
https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/WindowsTerminal/1.18.3181.0

Therefore I currently cannot update my Windows Terminal.

fyi: @consvc

**Comment by elisimpson** [comment]
Yep always fails at 95%

**Comment by Yousha** [comment]
Resolved by #120909 ?

**Comment by mdanish-kh** [comment]
> Yep always fails at 95%

Resolved through https://github.com/microsoft/winget-cli/pull/3975. Fix first appears in winget version v1.6.3482; You will need to upgrade if you are on a lower version. (Use https://aka.ms/getwinget to get the latest stable version of winget)

```
 ~ winget list terminal
Name                      Id                        Version     Available   Source
----------------------------------------------------------------------------------
Windows Terminal          Microsoft.WindowsTerminal 1.18.2822.0 1.18.3181.0 winget
Oh My Posh version 19.5.1 JanDeDobbeleer.OhMyPosh   19.5.1      19.5.2      winget
gsudo v2.4.0              gerardog.gsudo            2.4.0       2.4.2       winget
Alacritty                 Alacritty.Alacritty       0.12.2      0.13.0      winget
 ~ winget upgrade --id Microsoft.WindowsTerminal
Found Windows Terminal [Microsoft.WindowsTerminal] Version 1.18.3181.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
This package requires the following dependencies:
  - Packages
      Microsoft.UI.Xaml.2.8 [>= 8.2306.22001.0]
Successfully verified installer hash
Starting package install...
  ████████████████████████████▌   95%
Successfully installed. Restart the application to complete the upgrade.
 ~ winget -v
v1.6.3482
```
